permissions-revoke/idlharness.any.js

```diff
@@ -0,0 +1,27 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+idl_test(
+  ['permissions-revoke'],
+  ['permissions', 'html', 'dom'],
+  async idl_array => {
+    // Attempt to make a permissions object available via navigator.
+    try {
+      await navigator.permissions.query({ name: "geolocation" });
```

Comment:
Same doubt as in https://github.com/web-platform-tests/wpt/pull/21492#discussion_r375290299